ci/jenkins/test-mc.sh

```diff
@@ -246,8 +246,10 @@ function deliver_multicluster_controller {
 
     for kubeconfig in "${membercluter_kubeconfigs[@]}"
     do
-       ip=$(kubectl get nodes -o wide --no-headers=true ${EAST_CLUSTER_CONFIG} | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 == role {print $6}')
-       rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" ./multicluster/test/yamls/test-east-serviceexport.yml jenkins@["${ip}"]:"${WORKDIR}"/serviceexport.yml
+       IFS='/';arr=(${kubeconfig});unset IFS
```

Comment:
what is IFS for?

Reply:
IFS is `Internal Field Separator`, the format of `kubeconfig` is a path like `/var/lib/jenkins/.kube/east`, here we need the last value `east`, so I change IFS to '/', so we can get an array and retrieve last element.

Reply:
I think you should add some comments to explain this.

Reply:
done